React/wbooks/src/Components/BookCover.js

```diff
@@ -3,19 +3,20 @@ import PropTypes from 'prop-types';
 
 import defaultImg from '../assets/default_book.svg';
 
-function BookCover({ imageUrl, imgStyle }) {
+function BookCover({ imageUrl, imgStyle, defaultStyle }) {
   return imageUrl ? (
     <img className={imgStyle} alt="Book cover" src={imageUrl} />
   ) : (
     <div className={`${imgStyle} book-default-background`}>
-      <img alt="Default book cover" src={defaultImg} />
+      <img className={defaultStyle} alt="Default book cover" src={defaultImg} />
     </div>
   );
 }
 
 BookCover.propTypes = {
   imageUrl: PropTypes.string,
-  imgStyle: PropTypes.string
+  imgStyle: PropTypes.string,
```

Comment:
Required (or defaultProp `imageStyle: ''` if it's not). If it's undefined, you'll see `class="undefined book-default-background"` in the dom